projects/demo-cypress/src/tests/range.cy.ts

```diff
@@ -0,0 +1,275 @@
+import {Component, ElementRef, ViewChild} from '@angular/core';
+import {ComponentFixture} from '@angular/core/testing';
+import {FormControl, ReactiveFormsModule} from '@angular/forms';
+import {TuiRootModule} from '@taiga-ui/core';
+import {TuiKeySteps, TuiRangeComponent, TuiRangeModule} from '@taiga-ui/kit';
+
+describe('TuiRange', () => {
+    let component: TestComponent;
+    let fixture: ComponentFixture<TestComponent>;
+
+    @Component({
+        template: `
+            <tui-root>
+                <tui-range
+                    [formControl]="testValue"
+                    [keySteps]="keySteps"
+                    [max]="max"
+                    [min]="min"
+                    [segments]="segments"
+                    [step]="step"
+                ></tui-range>
+            </tui-root>
+        `,
+    })
+    class TestComponent {
+        @ViewChild(TuiRangeComponent, {static: true})
+        component!: TuiRangeComponent;
+
+        @ViewChild(TuiRangeComponent, {static: true, read: ElementRef})
+        el!: ElementRef<HTMLElement>;
+
+        testValue = new FormControl([3, 5]);
+        max = 11;
+        min = 1;
+        segments = 10;
+        step = 1;
+        keySteps: TuiKeySteps | null = null;
+    }
+
+    const keydownArrowLeft = new KeyboardEvent('keydown', {
+        bubbles: true,
+        key: 'ArrowLeft',
+    });
+
+    const keydownArrowRight = new KeyboardEvent('keydown', {
+        bubbles: true,
+        key: 'ArrowRight',
+    });
+
+    beforeEach(() =>
+        cy
+            .mount(TestComponent, {
+                imports: [TuiRootModule, TuiRangeModule, ReactiveFormsModule],
+            })
+            .then(wrapper => {
+                component = wrapper.component;
+                fixture = wrapper.fixture;
+            }),
+    );
+
+    it('The bar is filled from 20% to 40%', () => {
+        expect(getFilledRangeOffset(component).left).to.equal('20%');
+        expect(getFilledRangeOffset(component).right).to.equal('60%');
+    });
+
+    describe('Changing values', () => {
+        describe('Left point', () => {
+            it('Pressing the left arrow decreases the value by one step', () => {
+                getLeft(component)?.dispatchEvent(keydownArrowLeft);
```

Comment:
Could we use Cypress API ?
https://docs.cypress.io/api/commands/type

Something similar to the following lines
```ts
cy.get('[automation-id="tui-range__left"]').focus().type('{leftArrow}');
```

Reply:
Doesn't work 

<img width="385" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/15f62e17-4395-44c4-bf3a-4a38b36c074d">

Reply:
fixed